ext/zlib/zlib.c

```diff
@@ -930,71 +978,69 @@ PHP_FUNCTION(inflate_add)
 			break;
 
 		default:
-			zend_value_error(
-				"Flush mode must be ZLIB_NO_FLUSH, ZLIB_PARTIAL_FLUSH, ZLIB_SYNC_FLUSH, ZLIB_FULL_FLUSH, ZLIB_BLOCK or ZLIB_FINISH");
+			zend_argument_value_error(3, "must be one of ZLIB_NO_FLUSH, ZLIB_PARTIAL_FLUSH, ZLIB_SYNC_FLUSH, ZLIB_FULL_FLUSH, ZLIB_BLOCK, ZLIB_FINISH");
```

Comment:
I think the `or` should be retained here.

Reply:
Fixed these in the commit to master